IdentityCore/src/validation/MSIDAuthority.m

```diff
@@ -26,9 +26,52 @@
 //------------------------------------------------------------------------------
 
 #import "MSIDAuthority.h"
+#import "MSIDAadAuthorityCache.h"
+#import "MSIDAuthorityResolving.h"
+#import "MSIDAadAuthorityResolver.h"
+#import "MSIDAADAuthorityMetadataRequest.h"
+#import "MSIDDRSDiscoveryRequest.h"
+#import "MSIDWebFingerRequest.h"
+#import "MSIDAuthorityResolving.h"
+#import "MSIDAadAuthorityResolver.h"
+#import "MSIDB2CAuthorityResolver.h"
+#import "MSIDAdfsAuthorityResolver.h"
+#import "MSIDOpenIdConfigurationInfoRequest.h"
+
+static NSSet<NSString *> *s_trustedHostList;
+static NSCache <NSString *, MSIDOpenIdProviderMetadata *> *s_openIdConfigurationCache;
+
+// Trusted authorities
+NSString *const MSIDTrustedAuthority             = @"login.windows.net";
```

Comment:
Can you verify we have all authorities from here?

{
"preferred_network": "login.microsoftonline.com",
"preferred_cache": "login.windows.net",
"aliases": ["login.microsoftonline.com", "login.windows.net", "login.microsoft.com", "sts.windows.net"]
}, {
"preferred_network": "login.partner.microsoftonline.cn",
"preferred_cache": "login.partner.microsoftonline.cn",
"aliases": ["login.partner.microsoftonline.cn", "login.chinacloudapi.cn"]
}, {
"preferred_network": "login.microsoftonline.de",
"preferred_cache": "login.microsoftonline.de",
"aliases": ["login.microsoftonline.de"]
}, {
"preferred_network": "login.microsoftonline.us",
"preferred_cache": "login.microsoftonline.us",
"aliases": ["login.microsoftonline.us", "login.usgovcloudapi.net"]
}, {
"preferred_network": "login-us.microsoftonline.com",
"preferred_cache": "login-us.microsoftonline.com",
"aliases": ["login-us.microsoftonline.com"]
}]

Reply:
Should we add aliases to list of know authorities?